src/custom/constants/index.ts

```diff
@@ -22,7 +22,8 @@ export const PERCENTAGE_PRECISION = 2
 
 export const LONG_LOAD_THRESHOLD = 2000
 
-export const APP_ID = Number(process.env.REACT_APP_ID)
+export const METADATA_DIGEST_HEX =
```

Comment:
What about `APP_DATA_HASH`?

Reply:
I called digest, cause that was what IPFS inspector showed. Hex, is just so is clearly an hex and not base52 or other encoding.  Metadata, cause is order metadata. 

Anyways, using yours, not sure what is better, seams alright to me